src/Uploadcare/File.php

```diff
@@ -168,19 +168,20 @@ public function delete()
     return $this->api->__preparedRequest('file_storage', 'DELETE', array('uuid' => $this->uuid));
   }
 
+  
   /**
-   * Get url of original image
+   * Get url path of original file
    *
    * @param string $postfix
    * @return string
    */
-  public function getUrl($postfix = null)
+  public function getPath($postfix = null)
   {
-    $url = sprintf('%s/%s/', $this->api->getCdnUri(), $this->uuid);
-    if($this->default_effects) {
-      $url = sprintf('%s-/%s', $url, $this->default_effects);
+    $uuid = $this->uuid;
+    if ($this->default_effects) {
+      $path = sprintf('%s/%s/', $uuid, $this->default_effects);
```

Comment:
this results in incorrect path
https://uploadcare.com/documentation/cdn/#file-api